github-runners/k8s/dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:buster-slim
+
+ENV GITHUB_PAT ""
+ENV GITHUB_TOKEN ""
```

Comment:
These look like a very bad idea. Please don't compile tokens into container images and available to everything running it it, they should be passed in as arguments.